01 Displaying Data/displaying-data/src/app/app.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { Customer } from './models/customer.model';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent implements OnInit {
+  title = 'Displaying Data Demo';
+  private customers: Customer[];
```

Comment:
It is boilerplate for the next example. Open an issue, and we can discuss change this later.